testsuite/features/step_definitions/command_steps.rb

```diff
@@ -60,6 +60,31 @@
   node.run("rpm -q #{package}; test $? -ne 0")
 end
 
+Then(/^I wait for "([^"]*)" to be (uninstalled|installed) on "([^"]*)"$/) do |package, status, host|
```

Comment:
Rather "When" (there's no "should" in step title, it's rather an action than a test...)